### ISSUE TITLE
Instagram Gallery!!!

### ISSUE THREAD

**Issue by Jordan83902** [elementor/elementor#11315]
When will there be an Instagram Feed element?

Right now I am having to pay for a separate plugin to display an Instagram Feed.

Many other softwares/website builders have this element like Squarespace.

I feel that with the popularity of Instagram it would be something that literally everyone would at least consider using on their site. Whether it is a personal site or a company site.

Many thanks!

## Prerequisites

- [ ] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [ ] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by sunawang** [comment]
> When will there be an Instagram Feed element?
> 
> Right now I am having to pay for a separate plugin to display an Instagram Feed.
> 
> Many other softwares/website builders have this element like Squarespace.
> 
> I feel that with the popularity of Instagram it would be something that literally everyone would at least consider using on their site. Whether it is a personal site or a company site.
> 
> Many thanks!
> 
> ## Prerequisites
> * [ ]  I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
> * [ ]  The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )
> 
> **What problem is your feature request going to solve? Please describe.**
> A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
> 
> **Describe the solution you'd like**
> A clear and concise description of what you want to happen.
> 
> **Describe alternatives you've considered**
> A clear and concise description of any alternative solutions or features you've considered.
> 
> **Additional context**
> Add any other context or screenshots about the feature request here.

The Instagram widget seems won't be available in Elementor Pro in the near future. In the 4,000,0000

> When will there be an Instagram Feed element?
> 
> Right now I am having to pay for a separate plugin to display an Instagram Feed.
> 
> Many other softwares/website builders have this element like Squarespace.
> 
> I feel that with the popularity of Instagram it would be something that literally everyone would at least consider using on their site. Whether it is a personal site or a company site.
> 
> Many thanks!
> 
> ## Prerequisites
> * [ ]  I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
> * [ ]  The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )
> 
> **What problem is your feature request going to solve? Please describe.**
> A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
> 
> **Describe the solution you'd like**
> A clear and concise description of what you want to happen.
> 
> **Describe alternatives you've considered**
> A clear and concise description of any alternative solutions or features you've considered.
> 
> **Additional context**
> Add any other context or screenshots about the feature request here.

Have you tried PowerPack? It has a widget (available on the free version) to [display your Instagram feed in Elementor](https://www.wppagebuilders.com/elementor-add-ons-to-display-instagram-feed/).

**Comment by MichaelDarko** [comment]
+1

From my experience, more and more clients want some sort of Instagram feed on their website. There are good plugins for that, but it would be nice to have an Elementor widget for this.

**Comment by sunawang** [comment]
> +1
> 
> From my experience, more and more clients want some sort of Instagram feed on their website. There are good plugins for that, but it would be nice to have an Elementor widget for this.

It seems that Elementor won't develop a native Instagram widget in the near future. To celebrate 4M installations, the Elementor asked users a new feature they want and users prefer multi-step forms instead of Instagram widget. [https://elementor.com/blog/4-million-winning-feature/](https://elementor.com/blog/4-million-winning-feature/)

**Comment by rodolphebertozzo** [comment]
+1

**Comment by davidamarek** [comment]
+1

**Comment by fabianenzensperger** [comment]
+1. A social widget would be even better. Where to choose whether FB, IG or YT streams / videos.

**Comment by gerardreches** [comment]
Adding my vote for an instagram feed widget

**Comment by vgstef** [comment]
Yeah, more and more clients want to show Instagram on their website. Such a widget would be quite useful. I'd like to know if it's on the roadmap.

**Comment by 79ho3ein** [comment]
+1
